### PR TITLE
Add OpenFin blurb to implementations page

### PR DESCRIPTION
**Pull request by kriswest**
image

### CODE REVIEW

File: website/data/implementations.json [hunk 1]
```diff
@@ -84,5 +84,15 @@
...
+    },
+    {
+        "title": "OpenFin",
+        "publisher": "OpenFin Inc.",
+        "image": "/img/users/Openfin.png",
+        "infoLink": "https://openfin.co/",
+        "docsLink": "https://developers.openfin.co/of-docs",
+        "type": "platform-provider",
+        "badges": [ ],
+        "description": "<p>OpenFin founded the FDC3 standard in 2017 in collaboration with major banks and asset managers. Deployed at more than 2400 financial firms, OpenFin OS, allows applications from multiple providers to co-exist and interoperate on the desktop.</p><p>OpenFin OS consists of a secure, Chromium-based runtime called Container and a visual interface called Workspace. Workspace gets you running in no time, with themeable UI components for complex windowing, advanced search, actionable notifications and application discovery.</p><p>You can also build your own solutions using the OpenFin Container, native language adapters and APIs while remaining compatible with everything else built on OpenFin.</p>"
```
Comment (openfin-johans) [inline website/data/implementations.json]:
```suggestion
        "description": "<p>OpenFin founded the FDC3 standard in 2017 in collaboration with major banks and asset managers. Deployed at more than 2400 financial firms, OpenFin OS, allows applications from multiple providers to co-exist and interoperate on the desktop.</p><p>OpenFin OS consists of a secure, Chromium-based runtime called Container and a visual interface called Workspace. <a href='https://www.openfin.co/workspace/' alt=OpenFin Workspace'>Workspace</a> gets you running in no time, with themeable UI components for complex windowing, advanced search, actionable notifications and application discovery.</p><p>You can also build your own solutions using the <a href='https://www.openfin.co/container/' alt=OpenFin Container'>OpenFin Container</a>, native language <a href='https://www.openfin.co/container/adapters/' alt=OpenFin Native Adapters'>adapters</a> and APIs while remaining compatible with everything else built on OpenFin.</p>"
```

**Review by openfin-johans — CHANGES_REQUESTED** [review]
Added links to the website for various items.

File: website/data/implementations.json [hunk 2]
```diff
@@ -84,5 +84,15 @@
...
+    },
+    {
+        "title": "OpenFin",
+        "publisher": "OpenFin Inc.",
+        "image": "/img/users/Openfin.png",
+        "infoLink": "https://openfin.co/",
+        "docsLink": "https://developers.openfin.co/of-docs",
+        "type": "platform-provider",
+        "badges": [ ],
+        "description": "<p>OpenFin founded the FDC3 standard in 2017 in collaboration with major banks and asset managers. Deployed at more than 2400 financial firms, OpenFin OS, allows applications from multiple providers to co-exist and interoperate on the desktop.</p><p>OpenFin OS consists of a secure, Chromium-based runtime called Container and a visual interface called <a href='https://www.openfin.co/workspace/'>Workspace</a>. Workspace gets you running in no time, with themeable UI components for complex windowing, advanced search, actionable notifications and application discovery.</p><p>You can also build your own solutions using the <a href='https://www.openfin.co/container/'>OpenFin Container</a>, native language <a href='https://www.openfin.co/container/adapters/'>adapters</a> and APIs while remaining compatible with everything else built on OpenFin.</p>"
```
Comment (kriswest) [inline website/data/implementations.json:96]:
```suggestion
        "description": "<p>OpenFin founded the FDC3 standard in 2017 in collaboration with major banks and asset managers. Deployed at more than 2400 financial firms, OpenFin OS, allows applications from multiple providers to co-exist and interoperate on the desktop.</p><p>OpenFin OS consists of a secure, Chromium-based runtime called Container and a visual interface called <a href='https://www.openfin.co/workspace/' alt='OpenFin Workspace'>Workspace</a>. Workspace gets you running in no time, with themeable UI components for complex windowing, advanced search, actionable notifications and application discovery.</p><p>You can also build your own solutions using the <a href='https://www.openfin.co/container/' alt='OpenFin Container'>OpenFin Container</a>, native language <a href='https://www.openfin.co/container/adapters/' alt='OpenFin Native Adapters'>adapters</a> and APIs while remaining compatible with everything else built on OpenFin.</p>"
```

**Review by openfin-johans — APPROVED** [review]
Nice one - approved!

**Pull request merged**